### ISSUE TITLE
Version `v0.3.1` instead of `v0.3.2` shown in the Pod logs after updating to `v0.3.2`

### ISSUE THREAD

**Issue by LarsBingBong** [ori-edge/k8s_gateway#165]
Seeing:

![image](https://user-images.githubusercontent.com/80959443/197518086-0434035e-955c-4c4d-9128-ae70c8758d67.png)

Event though the container image is:

![image](https://user-images.githubusercontent.com/80959443/197518154-b821a648-ca83-42bf-9008-b373fd81224e.png)

That's throwing me off a bit. Shouldn't the version being printed by `v0.3.2` ??

Thank you

**Comment by networkop** [comment]
yep, it's a bug https://github.com/ori-edge/k8s_gateway/blob/v0.3.2/cmd/coredns.go#L19 :(
I really need to automate the release process. If anyone has any good suggestions/pointers, please let me know

**Comment by LarsBingBong** [comment]
@networkop good find! And thank you for spending the time. I'll give it a thought.

**Comment by LarsBingBong** [comment]
It didn't happen on the latest release. But, I see the approach is the same. You're just really skilled now at remembering to bump that constant 👍🏿
